monai/engines/trainer.py

```diff
@@ -85,6 +85,8 @@ class SupervisedTrainer(Trainer):
             new events can be a list of str or `ignite.engine.events.EventEnum`.
         event_to_attr: a dictionary to map an event to a state attribute, then add to `engine.state`.
             for more details, check: https://github.com/pytorch/ignite/blob/v0.4.4.post1/ignite/engine/engine.py#L160
+        decollate: whether decollate the batch-first data to a list of data after model computation, default to `True`.
+            if `False`, postprocessing may not work as all the transforms should apply on channel-first data.
```

Comment:
```suggestion
            if `False`, postprocessing may not work as the `monai.transforms` module assumes channel-first data.
```

Reply:
updated all the doc-strings in workflows.

Thanks.